discord/guild.py

```diff
@@ -3121,9 +3138,27 @@ async def ban(
                 "delete_message_seconds must be between 0 and 604800 seconds."
             )
 
-        await self._state.http.ban(
-            user.id, self.id, delete_message_seconds, delete_message_days, reason=reason
-        )
+        if len(users) == 1:
+            await self._state.http.ban(
+                users[0].id,
+                self.id,
+                delete_message_seconds,
+                delete_message_days,
+                reason=reason,
+            )
+        elif len(users) > 200:
+            raise ValueError("You may only bulk ban up to 200 members.")
```

Comment:
Make the error message more formal; e.g. the number of members that are banned must be less than or equal to 200